pilot/pkg/security/model/authentication_test.go

```diff
@@ -278,8 +278,7 @@ func TestApplyToCommonTLSContext(t *testing.T) {
 					{
 						Name: "file-cert:serverCertChain~serverKey",
 						SdsConfig: &core.ConfigSource{
-							InitialFetchTimeout: features.InitialFetchTimeout,
-							ResourceApiVersion:  core.ApiVersion_V3,
+							ResourceApiVersion: core.ApiVersion_V3,
```

Comment:
I am a bit confused why the test for `file-cert:serverCertChain~serverKey` is changing, but the test for `default` is not? Shouldn't it be the opposite?

Reply:
not sure. i guess the `ConstructSdsSecretConfigWithCustomUds` method constructs with the 0s InitialVerifyTimeout and its removed if the time is 0? not sure. is this an issue?